### PR TITLE
[DOCFIX]Doc fix for docs/cn/Debugging-Guide.md

### PR DESCRIPTION
**Pull request by Popeye13**

### CODE REVIEW

File: docs/cn/Debugging-Guide.md [hunk 1]
```diff
@@ -23,16 +23,13 @@ master和worker日志对于理解Alluxio master节点和worker节点的运行过
...
-```properties
-ALLUXIO_MASTER_JAVA_OPTS="-Xdebug -Xnoagent -Djava.compiler=NONE -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=6600 $ALLUXIO_JAVA_OPTS"
-
-ALLUXIO_WORKER_JAVA_OPTS="-Xdebug -Xnoagent -Djava.compiler=NONE -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=6601 $ALLUXIO_JAVA_OPTS"
-
-ALLUXIO_PROXY_JAVA_OPTS="-Xdebug -Xnoagent -Djava.compiler=NONE -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=6602 $ALLUXIO_JAVA_OPTS"
-```
+使用java远程调试技术可以简单、不修改源码的方式，进行源码级调试。你需要增加jvm 远程调试参数，启动调试服务。增加远程调试参数的方法有很多，你可以在需要调试的节点上，在命令行中或`alluxio-env.sh`中配置环境变量，增加如下配置属性。
 
++export ALLUXIO_WORKER_JAVA_OPTS="$ALLUXIO_JAVA_OPTS -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=6606"
```
Comment (apc999) [inline docs/cn/Debugging-Guide.md]: remove ++?

Comment (apc999) [inline docs/cn/Debugging-Guide.md]: missing
<pre>
```
</pre>
before line 28

**Comment by Popeye13** [comment]
@apc999 Thanks for your review, updated.

File: docs/cn/Debugging-Guide.md [hunk 2]
```diff
@@ -23,16 +23,13 @@ master和worker日志对于理解Alluxio master节点和worker节点的运行过
...
++export ALLUXIO_MASTER_JAVA_OPTS="$ALLUXIO_JAVA_OPTS -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=6607"
++export ALLUXIO_USER_JAVA_OPTS="$ALLUXIO_JAVA_OPTS -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=6609"
++```
++`suspend = y/n` 会决定JVM进程是否等待直至调试器连接。如果你希望在命令行中进行调试，设置`suspend = y`。否则，设置 `suspend = n` ，这样就可以避免不必要的等待时间。
 这样启动该节点上的master或者worker后，使用eclipse或intellij idea等java开发环境，新建java远程调试配置，设置调试主机名和端口号，然后启动调试连接。如果你设置了断点并且到达了断点处，开发环境会进入调试模式，可以读写当前现场的变量、调用栈、线程列表、表达式评估，也可以执行单步进入、单步跳过、恢复执行、挂起等调试控制。掌握这个技术使得日后的定位问题事半功倍，也会对调试过的代码上下文印象深刻。
```
Comment (Popeye13) [inline docs/cn/Debugging-Guide.md]: Thanks for your review, updated.

File: docs/cn/Debugging-Guide.md [hunk 3]
```diff
@@ -23,17 +23,19 @@ master和worker日志对于理解Alluxio master节点和worker节点的运行过
 
 Alluxio一般不在开发机上运行,这使得Alluxio的调试变得困难,我们通常会用"增加日志-编译-部署运行-查看日志"的方法来定位问题,而这种定位问题的效率比较低而且需要修改代码从新部署,这在有些时候是不允许的。
 
-使用java远程调试技术可以简单、不修改源码的方式，进行源码级调试。你需要增加jvm 远程调试参数，启动调试服务。增加远程调试参数的方法有很多，比较方便的一种方法是，在需要调试的节点上，修改`alluxio-env.sh`，增加如下配置属性。
+<<<<<<< HEAD
```
Comment (apc999) [inline docs/cn/Debugging-Guide.md]: @Popeye13  can you delete line 26 27 28 ? these are leftovers from git merge

**Comment by apc999** [comment]
@Popeye13 there is git merge conflict uncleaned. Otherwise it looks good to me.
Thanks!

**Comment by Popeye13** [comment]
@apc999  Thanks for your review, updated.

**Pull request merged**